Assets/Stronghold_State.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Stronghold_State : MonoBehaviour {
+
+    //Player color and name Properties
+    public Player player;
+
+    //Keep for testing Purposes of Mechanics *NOT NECESSARY*
+    public string color;
+    public int magnitude = 0;
+
+    //Scale of Strongholds
+    public int scale = 5;
+
+    //Rendering Component contains Material Color
+    public Renderer rend;
+
+    // Use this for initialization
+    void Start() {
+
+        //Get Rendering Component
+        rend = GetComponent<Renderer>();
+    
+        //Scale StrongHold to Required User Value
+        transform.localScale =new Vector3(scale,scale,scale);
+
+        //Instantiate Color and Magnitude
+        if (player != null)
+        {
+            Debug.Log("Changing Color");
+            rend.material.color = player.color;
+        }
+        else
+        {
+            Debug.Log("Player doesnt exist");
+        }
+
+    }
+	
+	// Update is called once per frame
+	void Update () {
```

Comment:
Since this isn't being used, I think we can remove it for now.

Reply:
Yeah we can replace it with onCollisionEnter() when we are implementing collisions.